Specs/DateTime/English/DateTimeModel.json

```diff
@@ -23722,5 +23738,229 @@
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "www.example.com/users/12345/archive-2018-09-23-page-01",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "2018-09-23",
+        "Start": 36,
+        "End": 45,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2018-09-23",
+              "type": "date",
+              "value": "2018-09-23"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "www.example2.com/data/report-20170302.html",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "20170302",
+        "Start": 29,
+        "End": 36,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2017-03-02",
+              "type": "date",
+              "value": "2017-03-02"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "DateTime search phrase 23.04.2022 14:55",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "23.04.2022 14:55",
+        "Start": 23,
+        "End": 38,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2022-04-23T14:55",
+              "type": "datetime",
+              "value": "2022-04-23 14:55:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "DateTime search phrase\n23.04.2022 14:55\n30.04.2022 12:04",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "23.04.2022 14:55",
+        "Start": 23,
+        "End": 38,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2022-04-23T14:55",
+              "type": "datetime",
+              "value": "2022-04-23 14:55:00"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "30.04.2022 12:04",
+        "Start": 40,
+        "End": 55,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2022-04-30T12:04",
+              "type": "datetime",
+              "value": "2022-04-30 12:04:00"
+            },
+            {
+              "timex": "2022-04-30T00:04",
+              "type": "datetime",
+              "value": "2022-04-30 00:04:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "DateTime search phrase\n23.04.2022 14:55\n30.04.2022 12:04 some text 2020",
```

Comment:
Could we add specs like this to a couple of the other languaes to make sure the issue doesn't happen there?

Reply:
added in NL|FR|DE|IT|PT|ES